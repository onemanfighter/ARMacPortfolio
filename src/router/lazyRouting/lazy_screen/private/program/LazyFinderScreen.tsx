import { lazy } from 'react';
import { LazyProvider } from '../../../component';

const Finder = lazy(
  () => import('../../../../../screens/private/program/Finder/Finder'),
);

const LazyFinderComponent = () => {
  return (
    <LazyProvider>
      <Finder />
    </LazyProvider>
  );
};

export { LazyFinderComponent };
