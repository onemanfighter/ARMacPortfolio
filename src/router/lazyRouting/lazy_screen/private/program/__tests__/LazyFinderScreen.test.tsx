import { render, screen, waitFor } from '@testing-library/react';
import { LazyFinderComponent } from '../LazyFinderScreen';

describe('LazyFinderComponent', () => {
  it('should render correctly to match snapshot', async () => {
    const { container } = render(<LazyFinderComponent />);

    await waitFor(() => expect(screen.getByText('Finder')).toBeDefined());
    expect(container).toMatchSnapshot();
  });
});
