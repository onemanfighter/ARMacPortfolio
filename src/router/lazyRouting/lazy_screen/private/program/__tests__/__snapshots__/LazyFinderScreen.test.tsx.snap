// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LazyFinderComponent should render correctly to match snapshot 1`] = `
<div>
  <div>
    <h1>
      Finder
    </h1>
  </div>
</div>
`;
